Remotion/Data/DomainObjects.RdbmsTools/Data.DomainObjects.RdbmsTools.csproj

```
@@ -20,7 +19,7 @@
     <ProjectReference Include="..\DomainObjects\Data.DomainObjects.csproj" />
     <ProjectReference Include="..\Core\Data.Core.csproj" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0-windows'">
```

```suggestion
  <ItemGroup Condition="'$(TargetFramework)'  == 'net48'">
```
To match Remotion.Core.csproj